dnf-behave-tests/features/repoinfo.feature

```diff
@@ -0,0 +1,121 @@
+Feature: Repoinfo
+
+
+Background: Using repositories dnf-ci-fedora and dnf-ci-thirdparty-updates
+  Given I use repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-thirdparty-updates"
+    And I use repository "dnf-ci-fedora-updates" with configuration
+        | key     | value |
+        | enabled | 0     |
+    And I use repository "dnf-ci-thirdparty" with configuration
+        | key     | value |
+        | enabled | 0     |
+
+@bz1793950
+Scenario: Repolist without arguments
+   When I execute dnf with args "repoinfo"
+   Then the exit code is 0
+    And stdout matches line by line
+"""
+
+Repo-id            : dnf-ci-fedora
+Repo-name          : dnf-ci-fedora test repository
+Repo-revision      : 1550000000
+Repo-updated       : .*
+Repo-pkgs          : 289
+Repo-available-pkgs: 289
+Repo-size          : 2.0 M
+Repo-baseurl       : .*/opt/behave/fixtures/repos/dnf-ci-fedora
```

Comment:
The Repo-baseurl strongly depends on the directory the test was run from. Currently the test passes when run using dnf-testing.sh in the container. But if you ran it directly using behave, the path would change - for example I have "Repo-baseurl       : file:///home/mblaha/src/ci-dnf-stack/dnf-behave-tests/fixtures/repos/dnf-ci-fedora" there.
The same for following tests.